packages/dashboard/README.md

```diff
@@ -146,6 +146,43 @@ In general try to append any new types to these files before creating a new `*.t
 
 ---
 
+# Types vs Interfaces
+
+With the latest typescript version, functionalities and capabilities of both types and interface become more similar. Hence for dashboard project, here are the cases when to use types or interfaces
+
+Do use interface to typecheck react proptypes object, you can also use interface when defining new object for classes because you can extend or implement an interface.
+```tsx
+
+const About: React.FC<IAboutProps> = ({ title, description }) => {
+    ...
+};
+
+interface IAboutProps {
+  title: string;
+  description: number;
+};
+```
+
+Do use types to typecheck a function. aside from the similar ability it possess with interface, you can use types to typecheck the argument and the return type of a function.
+
+```ts
+type TStatusShape = {
```

Comment:
I know we haven't decided yet if we should use interfaces or types, but we also need to decide if we add `T` & `I` prefixes or not cause I see on some files you added them and on some didn't.

Reply:
added the detail in readme
Tl;dr
for react props, i prefer to have T prefix to differentiate with the component name itself, e.g Button -> TButton . for other types, we dont need to as bala suggested type itself symbolizes that its a type